docs/guides/redis/reconfigure-tls/standalone.md

```diff
@@ -264,18 +258,6 @@ OK
 
 Now we are going to rotate the certificate of this database. First let's check the current expiration date of the certificate.
```

Comment:
First let's check the current expiration date of the certificate.-> we have removed the checking part